wolfcrypt/test/test.c

```diff
@@ -15729,6 +15729,9 @@ int ecc_test_buffers(void) {
     int verify = 0;
     word32 x;
 
+	memset(&cliKey, 0, sizeof(ecc_key));
+	memset(&servKey, 0, sizeof(ecc_key));
```

Comment:
Would you mind changing this to `XMEMSET` for portability. Also please use 4 spaces, not hard tabs. Thanks!